src/gtk_helpers.rs

```diff
@@ -145,3 +152,158 @@ where
         snapshot.to_paintable(None)
     }
 }
+
+// Calculate pixel width of a string given the label it's displayed in
```

Comment:
Function and method comments should be in rustdoc format, and use 3rd person singular form (ie it should answer the question "what does it do?"):

```suggestion
/// Calculates pixel width of a string given the label it's displayed in.
```